Echo.ProcessJS/scripts/process.js

```diff
@@ -278,6 +278,7 @@ var Process = (function () {
     var makeDoneThunk = function (value, ctx) {
         var thunk = makeThunk(ctx);
         setImmediate(function () { thunk.doneFn(value); });
+        thunk.wait = function (f) { f(value); };
```

Comment:
Couldn't think be: 
```
thunk.wait = function () { return value; };
```
I haven't studied the implications of that yet, but it *could* allow for `var res = ask(...).wait()` right?

Reply:
Yeah that makes more sense, changed it